sections/dom.include

```diff
@@ -604,140 +604,75 @@
-    in HTML.
-  </div>
-
-  Authors must not use elements, attributes, or attribute values for purposes other than their
-  appropriate intended semantic purpose, as doing so prevents software from correctly processing the
-  page.
+</div>
+
+This semantic information is critical to assistive technologies. For example, a screen reader will query the browser for semantic information and use that information to present the document or application in synthetic speech.
+
+In some cases assistive technologies use semantic information to provide additional functionality. A speech recognition tool might provide a voice command for moving focus to the start of the main element for example.
+
+When the appropriate HTML element or attribute is not used, it deprives HTML processors of valuable semantic information.
+
+<div class="example">
+<p>In this example the visual manifestation of the document may look the part, but because the appropriate HTML elements have not been used there is little semantic information available to browsers, search engines or assistive technologies.</p>
+
+<pre highlight="html">
+    &lt;!doctype html&gt;
+&lt;html lang="en"&gt;
+  &lt;head&gt;
+    &lt;title&gt;Favorite books&lt;/title&gt;
+  &lt;/head&gt;
+  &lt;body&gt;
+    &lt;div class="header"&gt;
+      &lt;img src="logo.png" alt="Favorite books logo"&gt;
+    &lt;/div&gt;
+    &lt;div class="main"&gt;
+      &lt;span class="largeHeading"&gt;Favorite books&lt;/span&gt;
+      &lt;p&gt;These are a few of my favorite books.&lt;/p&gt;
+      &lt;span class="smallHeading"&gt;The Belgariad&lt;/span&gt;
+      &lt;p&gt;Five books by David and Leigh Eddings.&lt;/p&gt;
+      &lt;span class="smallHeading"&gt;The Hitchhiker's Guide to the Galaxy&lt;/span&gt;
+      &lt;p&gt;A trilogy of five books by Douglas Adams.&lt;/p&gt;
+    &lt;/div&gt;
+  &lt;/body&gt;
+&lt;/html&gt;
+</pre>
+</div>
+
+A document can change dynamically whilst it is being processed. Scripting and other mechanisms can be used to change attribute values, text, or the entire document structure. The semantics of a document are therefore based on the document's state at a particular instance in time, but may also change in response to external events. User agents <em>must</em> update their presentation of the document to reflect these changes.
 
   <div class="example">
-    <pre class="bad" highlight="html">
-      &lt;body&gt;
-        &lt;h1&gt;ACME Corporation&lt;/h1&gt;
-        &lt;h2&gt;The leaders in arbitrary fast delivery since 1920&lt;/h2&gt;
-        ...
-    </pre>
-  </div>
+  <p>In this example the <{audio}> element is used to play a music track. The <dfn attribute for="Document"><code>controls</code></dfn> attribute is used to show the user agent player, and as the music plays the controls are updated to indicate progress. The available semantic information is updated in response to these changes.</p>
```

Comment:
The `controls` attribute is not defined on the `Document` element. I think this should be changed to {{mediaelements/controls}} to link to the dfn for controls for all media elements.